build.gradle

```diff
@@ -57,14 +57,18 @@ jmh {
     resultFormat = "TEXT"
 }
 
+ext {
+    kafkaVersion = "2.4.0"
```

Comment:
IMHO I would prefer a separate PR for minimum Kafka version

Reply:
It's an integral part of this change, because Kafka 2.4 brings the new API. It should be here.

Reply:
kk